continuous_integration/circle/install.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+
+# -- Install Miniconda
+MINICONDA=Miniconda3-latest-Linux-x86_64.sh
+wget https://repo.continuum.io/miniconda/$MINICONDA -O miniconda.sh
+bash miniconda.sh -b -p $HOME/miniconda
+source $HOME/miniconda/etc/profile.d/conda.sh
+
+# -- Install Yarn
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update && sudo apt-get install yarn
+export PATH="$PATH:/opt/yarn-1.16.0/bin"
+
+# -- Make new conda environment with required Python version
+conda create -y -n test python=$PYTHON_VERSION
+conda activate test
+
+# Install dependencies
+conda install -q -y -c spyder-ide --file requirements/conda.txt
+
+# Install test dependencies
+conda install -q -y -c spyder-ide --file requirements/tests.txt
+
+# Install qt
+sudo apt-get install build-essential
+
+sudo apt-get install qtcreator
+
+sudo apt-get install qt5-default
```

Comment:
Hi @steff456. Question. Why are we using apt instead of conda?

Reply:
I'm removing that lines and installing qt by pip

Reply:
Oh yeah, this is not necessary at all.